dist/functions.setup-appliance.sh

```diff
@@ -647,6 +642,21 @@ function prepare_os_settings {
         OBS_SIGND=signd
         SIGND_BIN="/usr/sbin/signd"
       ;;
+      ubuntu|debian)
+        MYSQL_SERVICE=mariadb
+        HTTPD_SERVICE=apache2
+        HTTPD_USER=www-data
+        HTTPD_GROUP=www-data
+        PASSENGER_CONF=/etc/$HTTPD_SERVICE/mods-available/passenger.conf
+        TRUST_ANCHORS_DIR=/usr/local/share/ca-certificates
```

Comment:
This is the wrong location. `/etc/ssl/certs` is the correct location.

Reply:
I think that's the way debian family handles it, the man of `update-ca-certificates` says:
```
Furthermore  all  certificates  with a .crt extension found below /usr/local/share/ca-certificates are also included as implicitly trusted.
```